src/Components/NavBar/Navbar.tsx

```diff
@@ -7,24 +7,37 @@ import { styled } from "@mui/material/styles";
 import SearchIcon from "@mui/icons-material/Search";
 import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
 
-export const StyledTypography = styled(Typography)(({ theme }) => ({
-  fontSize: "1rem",
-  fontWeight: "200",
-  textAlign: "center",
-  color: "#000",
-  fontFamily: "'Poppins', sans-serif",
-}));
+type NavProps = {
+  black?: boolean;
+  NumItems?: number;
+};
+const Child = ({ black, NumItems }: NavProps) => {
```

Comment:
Please consider using a more descriptive name for your component, **Child** of what ?